docs/docs/forms.mdx

```diff
@@ -32,12 +32,34 @@ The following example form `restaurant_form` will fill the slot
 ```yaml-rasa
 forms:
   restaurant_form:
-    cuisine:
-      - type: from_entity
-        entity: cuisine
-    num_people:
-      - type: from_entity
-        entity: number
+    required_slots:
+        cuisine:
+          - type: from_entity
+            entity: cuisine
+        num_people:
+          - type: from_entity
+            entity: number
+```
+
+You can define a list of intents to ignore for the whole form under the 
+`ignored_intents` key. Intents listed under `ignored_intents` will be added to the `not_intent` key of each required slot in the form.
```

Comment:
```suggestion
`ignored_intents` key. Intents listed under `ignored_intents` will be added to the `not_intent` key of each slot mapping in the form.
```

This is my bad, I suggested `required slot` in the first place, but on re-reading it it actually applies per mapping, not just per slot

Reply:
Thank YOU for re reading it!!! 💯